elegant-business/sass/site/header/_site-header.scss

```diff
@@ -2,7 +2,6 @@
 
 .site-header {
 	padding: 1.5em 1em 1em;
-	text-align: center;
```

Comment:
again these changes are not on the compiled css, we should discard these (https://github.com/Automattic/themes/commit/82beeb1fe7856a8d87f1fc8d783fa810b78aa619#diff-8da09c1b905c542f3928d6274720572d54ada9102e85337eda0ca8e93a85a33f)